controller/api/api.go

```diff
@@ -612,8 +612,15 @@ func (a *Api) scaleContainer(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if err := a.manager.ScaleContainer(containerId, numInstances); err != nil {
-		http.Error(w, err.Error(), http.StatusInternalServerError)
+	if errors := a.manager.ScaleContainer(containerId, numInstances); len(errors) > 0 {
+		errStrs := []string{}
+		for _, e := range errors {
+			if e != nil {
+				errStrs = append(errStrs, e.Error())
+			}
+		}
+		println(strings.Join(errStrs, ","))
+		http.Error(w, strings.Join(errStrs, ";"), http.StatusInternalServerError)
```

Comment:
didn't see the join -- this would be ok.  We should use `log.Error` instead of the `println`.  Also, perhaps a richer data type to return to the client: `{"errors":[]}` or something.